src/sagas/api.js

```diff
@@ -0,0 +1,128 @@
+import { 
+    INITIAL_CANVAS_DATA, 
+    INITIAL_CANVAS_WIDTH, 
+    INITIAL_CANVAS_HEIGHT } from '../constants/canvas'
+const HEADERS = {
```

Comment:
I see we already have this constant here; I made a comment elsewhere about creating a headers object. Maybe we can move this constant into the constants folder and import it anywhere we need it from there?

Reply:
Moved constant header to its own namespace.